File: IOT/anshulgurjar53_AnshulSinghGurjar_2024it1193_2/FunctionAndArrays/FindElementInArray.java
import java.io.*;
import java.util.*;

public class Main{

public static void main(String[] args) throws Exception {
    // write your code here
    Scanner sc= new Scanner(System.in);
    int n=sc.nextInt();int i;
    int a[]=new int[n];
    for( i=0;i<n;i++)
    {
        a[i]=sc.nextInt();
    }
    int d=sc.nextInt();
    for(i=0;i<n;i++)
    {
        if(a[i]==d)
        break;
    }
    if(i==n)
    System.out.println(-1);
    else
    System.out.println(i);
 }

}

File: Web/sneha-jais_snehaJaiswal_2024cs1084_2/FunctionAndArrays/find_element_in_array.cpp
#include<iostream>
using namespace std;


int main(){
    //write your code here
    int k=-1;
    int n;
    cin>>n;
    int arr[n];
    for(int i=0; i<n; i++)
    cin>>arr[i];
    int d;
    cin>>d;
    
    for(int i=0; i<n; i++){
        if(arr[i]==d)
         k=i;
 }
    cout<<k;    
    return 0;
}